literature.bib

```diff
@@ -1693,6 +1757,23 @@ @inproceedings{SL:VaMoS21
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2020 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@article{SC:OOPSLA20,
+	author = {Shahin, Ramy and Chechik, Marsha},
+	title = {{Automatic and Efficient Variability-Aware Lifting of Functional Programs}},
+	year = 2020,
+	publisher = ACM,
+	address = NY,
+	volume = {4},
+	number = {OOPSLA},
+	url = {https://doi.org/10.1145/3428225},
```

Comment:
I think we remove the URL if its basically just the DOI

Reply:
removed url